cicecore/cicedyn/general/ice_forcing.F90

```diff
@@ -2237,30 +2238,42 @@ subroutine JRA55_files(yr)
       ! cnt represents the possible file format options and steps thru them until one is found
       exists = .false.
       cnt = 1
-      do while (.not.exists .and. cnt <= 6)
+      do while (.not.exists .and. cnt <= 8)
+
+         ! search for jra55 file with creation date (jra55date) in filename first
          if (cnt == 1) uwind_file = trim(atm_data_dir)//'/'//trim(atm_data_type_prefix)// &
+                                    '/8XDAILY/'//trim(atm_data_type_prefix)//'_'//trim(atm_data_date)//'_'// &
+                                    trim(grd)//'_03hr_forcing_2005.nc'
+
+         if (cnt == 2) uwind_file = trim(atm_data_dir)//                                  &
+                                    '/8XDAILY/'//trim(atm_data_type_prefix)//'_'//trim(atm_data_date)//'_'// &
+                                    trim(grd)//                                              '_03hr_forcing_2005.nc'
+
+         ! now search for files without jra55date in filename.
```

Comment:
I don't think we should ever search for file without "atm_data_date".  The code will read the old files.  That means we need to set atm_data_date='' in ice_in, and it needs to be set to '_20230919' for tx1.